docs/Contributing/design-qa-considerations.md

```diff
@@ -0,0 +1,79 @@
+# Design / QA considerations
+
+This is meant to be a helpful checklist of 'events' or 'transactions' to remember to think about while designing or testing new features or bugs to make sure edge cases are caught sooner rather than later. Please feel free to add more if any are missing
+
+## User
+
+- Create User
+- Remove User
+- Update User permissions
+
+## Team
+
+- Create Team
+- Remove Team
+- Transfer Host into this team
+- Transfer Host out of this team
+
+## MDM
+
+- Turn MDM On
+- Turn MDM Off
+- Enable Disk Encryption
+- Disable Disk Encryption
+- Add ABM Token
+- Remove ABM Token
+- Add VPP Token
+- Remove VPP Token
+- Add minimum version OS Updates
+- Remove minimum version OS Updates
+- Add Profile
+- Remove Profile
+- Add Bootstrap Package
+- Remove Bootstrap Package
+- Setup Experience Software / scripts
+
+## Software
+
+- Add Software to team
```

Comment:
Just to make it explicit so folks think about all possibilities, "Add FMA/VPP/Custom Software to team", "Remove /Edit...".